Sources/NIO/Codec.swift

```diff
@@ -448,6 +453,9 @@ extension ByteToMessageHandler {
         var allowEmptyBuffer = decodeMode == .last
         while (self.state.isActive && self.removalState == .notBeingRemoved) || decodeMode == .last {
             let result = try self.withNextBuffer(allowEmptyBuffer: allowEmptyBuffer) { decoder, buffer in
+                if self.maximumBufferSize > 0, buffer.readableBytes >= self.maximumBufferSize {
```

Comment:
shouldn't this be `> self.maximumBufferSize`?

Reply:
depends on how you think of max? can go either way. lmk if you prefer = to be fine

Reply:
My view is the the maximum is an inclusive bound: if told me in a conversation that your maximum buffer size is 1024 bytes, I would expect you to accept a 1024 byte buffer.